compiler-plugin-tests/src/test/resources/ballerina_sources/generator_tests/23_sdl_file_generation/schema.graphql

```diff
@@ -0,0 +1,64 @@
+schema {
```

Comment:
Should we add this here? Even in that case, is this the correct way of showing this?

Reply:
This should be added only when the schema description is `null` and not using common names(Query, Mutation, Subscription) for root operations. Fixed with https://github.com/ballerina-platform/module-ballerina-graphql/pull/955/commits/1d66e9e7ed8248123bfe04e29a59a9511db48f9f

Reply:
If the schema description is `null`, what's the point of having a separate `schema` type? IIUC, the newest spec introduced the `__Schema` introspection type to include the schema description.
In Ballerina, we don't allow custom type names for root operations. So that does not apply to us, right?

Reply:
Removed since this is not applicable for the Ballerina GraphQL.